libcloud/compute/drivers/kubevirt.py

```diff
@@ -179,125 +245,103 @@ def destroy_node(self, node):
-                             -name: The name of the disk configuration
-                             -claim_name: the name of the
-                                         Persistent Volume Claim
-
-                            If you wish a new Persistent Volume Claim can be
-                            created by providing the following:
-                            required:
-                            -size: the desired size (implied in GB)
-                            -storage_class_name: the name of the storage class to # NOQA
-                                               be used for the creation of the
-                                               Persistent Volume Claim.
-                                               Make sure it allows for
-                                               dymamic provisioning.
-                             optional:
-                            -access_mode: default is ReadWriteOnce
-                            -volume_mode: default is `Filesystem`,
-                                         it can also be `Block`
-
-        :type ex_disks: `list` of `dict`. For each `dict` the types
-                            for its keys are:
-                            -bus: `str`
-                            -device: `str`
-                            -disk_type: `str`
-                            -name: `str`
-                            -claim_name: `str`
-                            (for creating a claim:)
-                            -size: `int`
-                            -storage_class_name: `str`
-                            -volume_mode: `str`
-                            -access_mode: `str`
-
-        :param ex_network: Only the pod type is supported, and in the
-                               configuration masquerade or bridge are the
-                               accepted values.
-                               The parameter must be a tuple or list with
-                               (network_type, interface, name)
-        :type ex_network: `iterable` (tuple or list) [network_type, interface, name]
-                      network_type: `str` | only "pod" is accepted atm
-                      interface: `str` | "masquerade" or "bridge"
-                      name: `str`
-
-        :param ports: A dictionary with keys: 'ports_tcp' and 'ports_udp'
-                      'ports_tcp' value is a list of ints that indicate
-                      the ports to be exposed with TCP protocol,
-                      and 'ports_udp' is a list of ints that indicate
-                      the ports to be exposed with UDP protocol.
-        :type  ports: `dict` with keys
-                      'ports_tcp`: `list` of `int`
-                      'ports_udp`: `list` of `int`
+        # check if new node is present
+        # But why not just use the resp from the POST request?
+        # Or self.get_node()?
+        # I don't think a for loop over list_nodes is necessary.
```

Comment:
Yeah, I agree, either using `list_nodes()` or `get_node()` which calls `list_nodes()` underneath is not really great and efficient...